web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -40,11 +44,23 @@ accelerate PyArrow-backed data in pandas, notibly string and datetime types.
 
 As of pandas 2.0, one can feasibly utilize PyArrow as an alternative data representation to NumPy with advantages such as:
 
-1. Consistent `NA` support for all data types
-2. Broader support of data types such as `decimal`, `date` and nested types
+1. Consistent `NA` support for all data types;
+2. Broader support of data types such as `decimal`, `date` and nested types;
+3. Better interoperability with other dataframe libraries based on Arrow.
+
+## Motivation
+
+While all the functionality described in the previous paragraph is currently optional, PyArrow has significant
+integration into many areas of pandas. With our roadmap noting that pandas strives for better Apache Arrow
+interoperability [^1] and many projects [^2], within or beyond the Python ecosystem, adopting or interacting with
+the Arrow format, making PyArrow a required dependency provides an additional signal of confidence in the Arrow
+ecosystem (as well as improving interoperability with it).
+
+### Immediate User Benefit 1: pyarrow strings
 
 Currently, when users pass string data into pandas constructors without specifying a data type, the resulting data type
-is `object`. With pyarrow string support available since 1.2.0, requiring pyarrow for 3.0 will allow pandas to default
+is `object`, which has horrendous memory and performance implications.
```

Comment:
"horrendous" is a bit of a strong word.  I think it would be better to say "which has poor memory usage and performance as compared to pyarrow strings."

Reply:
IMO horrendous is appropriate. I observed up to 5x more memory consumption with NumPy object and performance is just terrible if you compare head-to-head.

I did a bunch of benchmarks here: https://medium.com/towards-data-science/utilizing-pyarrow-to-improve-pandas-and-dask-workflows-2891d3d96d2b

Just one example: Avoiding the Arrow strings -> object conversion in read_parquet reduces the runtime by 2/3 ( I am talking about the runtime of the whole ``read_parquet`` call, e.g. the conversion actually takes __at least twice as long__ as I/O.

"poor" isn't strong enough imo 😅

Reply:
> "poor" isn't strong enough imo 😅

Fair enough, but it just didn't feel right to have the word "horrendous" in a PDEP.  The reality is that we have been living with this bad memory and performance for years, so we only know it is horrendous because of the comparison.   So maybe if you said "which has significantly much worse memory usage and performance as compared to pyarrow strings." it would feel right.

Reply:
+1 to `"which has significantly much worse memory usage and performance as compared to pyarrow strings."`

Reply:
Updated with Matts suggestion